explorer/src/constants/routes.ts

```diff
@@ -1,8 +1,11 @@
 export enum Routes {
-  nova = 'nova',
   consensus = 'consensus',
-  leaderboard = 'leaderboard',
+  farming = 'farming',
   staking = 'staking',
+  leaderboard = 'leaderboard',
+  domain = 'domain',
```

Comment:
I would make this plural
```suggestion
  domains = 'domains',
```